content/applications/finance/fiscal_localizations/peru.rst

```diff
@@ -942,3 +947,125 @@ a scheduled action, but you can also send each invoice manually if needed.
+update <general/install>` all Peruvian modules (`l10n_pe`).
+
+Most importantly, install the :guilabel:`Peru - Stock Reports` (`l10n_pe_reports_stock`) module.
+
+Update relevant fields
+**********************
+
+Once the **Peru - Stock Reports** module is installed, consider updating and completing the fields
+for:
+
+- :ref:`Products <peru/reports-ple-products>`
+- :ref:`Warehouses <peru/reports-ple-warehouses>`
+- :ref:`Inventory transfers <peru/reports-ple-transfers>`
+
+.. _peru/reports-ple-products:
+
+Products
+^^^^^^^^
+
+There are a few ways to configure a product for |PLE| reporting:
+
+- **Type of existence**: Navigate to :menuselection:`Sales --> Products --> Products` and select a
+  product. In the :guilabel:`Accounting` tab, select the :guilabel:`Type of Existence` according to
+  |SUNAT|'s table 5 for inventory reporting.
+
+  .. image:: peru/l10n-pe-type-of-existence.png
+     :alt: Select the Type of Existence for a product.
+
+- **Automatic inventory valuation**: For storable goods (:dfn:`products with tracked inventory`),
+  it's important to use :doc:`automatic inventory valuation
+  <../../inventory_and_mrp/inventory/product_management/inventory_valuation/inventory_valuation_config>`.
+  Once automatic inventory valuation is turned on, this valuation method can be turned on for
+  a product's :ref:`Product Category <inventory/warehouses_storage/valuation-on-product-category>`.
+
+- **Costing method:** Storable goods also need a :doc:`costing method
+  <../../inventory_and_mrp/inventory/product_management/inventory_valuation/inventory_valuation_config>`
+  that is **not** set to `Standard Price`. This is important as the journal entries generated from
+  stock moves feed the |PLE| reports.
+
+.. _peru/reports-ple-warehouses:
+
+Warehouses
+^^^^^^^^^^
+
+When :doc:`setting up a warehouse
+<../../inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses>`, the
+:guilabel:`Annex Establishment Code` field is important for reporting. This code acts as a unique
+ID and should be unique to each warehouse. The number should only be a numeric combination,
+containing between 4 to 7 digits.
+
+.. image:: peru/l10n-pe-annex-establishment-code.png
+   :alt: Choose a unique ID between 4-7 numeric digits.
+
+.. _peru/reports-ple-transfers:
+
+Inventory transfers
+^^^^^^^^^^^^^^^^^^^
+
+Transferring inventory is a key process captured in the |PLE| 12.1 and |PLE| 13.1 reports.
+:doc:`Inventory transfers <../../inventory_and_mrp/inventory/shipping_receiving/daily_operations>`
+include both incoming and outgoing shipments.
+
+When validating an inventory transfer (either on a warehouse receipt or delivery order), select the
+:guilabel:`Type of Operation (PE)` performed according to |SUNAT|'s table 12 for permanent inventory
+reporting.
+
+.. image:: peru/l10n-pe-type-of-operation-pe.png
+   :alt: Select a Type of Operations from SUNAT's table 12.
```

Comment:
Hi @dade-odoo users tend to not find these field easily, that's why the addition on visually help them know where to find it, also for the leads to know we do cover those mandatory fields.
But feel free to skip it, cheers!

Reply:
I understand wanting to make it easy for users, but we avoid screenshots that just show "hey this field exists/is located here/has these options" because these things frequently change, and then the user can be even more confused if their DB doesn't match the screenshot. Your text is clear enough to me in explaining that we cover those mandatory fields, so I do think we should get rid of this one too.